practice-two/src/index.html

```diff
@@ -178,20 +178,20 @@ <h2 class="references-section-heading">Referanslarımız</h2>
                 <p class="references-section-subheading">Bize güvenen ve beraber çalıştığımız iş ortaklarımız</p>
             </div>
             <div class="references-section-list-content">
-                <div class="references-section-list first-content">
+                <div>
```

Comment:
Do we need this `div`?

Reply:
I use this div to group cards for tablet and desktop screens